CMSgov/easi-app
# [EASI-3244] Confirm lcid

# EASI-3244

[Figma](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=2426%3A12753&mode=design&t=lycOMDCcriResAXd-1)

## Changes and Description

- Updates Issue LCID form for Confirm LCID action
  - Hides LCID radio and dropdown fields
  - Populates LCID fields with values from Issue LCID form
- Unit test



## How to test this change

1. Log in with `enableItGovV2` flag turned on
2. Issue LCID
    - Go to /governance-review-team/:systemId/actions
    - Select "Issue a decision or close this request"
    - Select "Issue a Life Cycle ID"
    - Complete form to issue LCID
3. Confirm LCID
    - Go back to  /governance-review-team/:systemId/actions
    - Select "Change decision or re-open this request"
    - Select "Confirm current decision (Issue a Life Cycle ID)"
    - Confirm that "Current LCID" box is displayed
    - Confirm that LCID radio and dropdown fields are hidden
    - Test form functionality
      - Success message should be the same as Issue Lcid (confirmed with Zoe)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
